corio.py

```diff
@@ -237,13 +240,24 @@ def log_status(parsed_input: dict, corio_start_time: datetime.time, test_failed:
                         input_dict["RESULT_UPDATE"] = f"Passed at {pass_time}"
                     else:
                         input_dict["RESULT_UPDATE"] = "In Progress"
+                    if input_dict["RESULT_UPDATE"] == "In Progress":
+                        # Report failure and update status.
+                        if terminated_tests:
+                            if input_dict["TEST_ID"] in terminated_tests:
+                                input_dict["RESULT_UPDATE"] = "Fail"
```

Comment:
can we add the time stamp for fail/aborted

Reply:
Total execution time updated at line no 249.